sugar/chains.py

```diff
@@ -586,13 +704,45 @@ def __init__(self, **kwargs): super().__init__(make_op_chain_settings(**kwargs),
 
 
 # %% ../src/chains.ipynb 14
+_base_settings = make_base_chain_settings()
+
 class BaseChainCommon():
-    usdc: Token = Token(token_address='0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913', symbol='USDC', decimals=6, listed=True, wrapped_token_address=None)
-    aero: Token = Token(token_address='0x940181a94A35A4569E4529A3CDfB74e38FD98631', symbol='AERO', decimals=18, listed=True, wrapped_token_address=None)
-    eth: Token = Token(token_address='ETH', symbol='ETH', decimals=18, listed=True, wrapped_token_address='0x4200000000000000000000000000000000000006')
+    usdc: Token = Token(chain_id=_base_settings.chain_id, chain_name=_base_settings.chain_name,
+                        token_address='0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913', symbol='USDC', decimals=6, listed=True, wrapped_token_address=None)
+    aero: Token = Token(chain_id=_base_settings.chain_id, chain_name=_base_settings.chain_name,
+                        token_address='0x940181a94A35A4569E4529A3CDfB74e38FD98631', symbol='AERO', decimals=18, listed=True, wrapped_token_address=None)
+    eth: Token = Token(chain_id=_base_settings.chain_id, chain_name=_base_settings.chain_name,
+                       token_address='ETH', symbol='ETH', decimals=18, listed=True, wrapped_token_address='0x4200000000000000000000000000000000000006')
     
 class AsyncBaseChain(AsyncChain, BaseChainCommon):
     def __init__(self, **kwargs): super().__init__(make_base_chain_settings(**kwargs), **kwargs)
 
 class BaseChain(Chain, BaseChainCommon):
     def __init__(self, **kwargs): super().__init__(make_base_chain_settings(**kwargs), **kwargs)
+
+# %% ../src/chains.ipynb 16
+class UniChainCommon():
```

Comment:
Should we include all supported chains maybe? These won't change often, but we can dynamically prep the relevant imports. Wdyt?